pyproject.toml

```diff
@@ -101,15 +102,8 @@ ignore = [
 "distributed/shuffle/tests/test_shuffle.py" = ["F601"]
 "distributed/utils.py" = ["F821"]
 
-[tool.isort]
-sections = ["FUTURE", "STDLIB", "THIRDPARTY", "DISTRIBUTED", "FIRSTPARTY", "LOCALFOLDER"]
```

Comment:
It looks like ruff supports sections, can you make sure we are configuring things the same way? We use these sections specifically to improve readability.

I would suggest going through each setting we had with isort and checking if you can replicate the same setting with ruff.

Reply:
Nevertheless, I have added this section in ruff settings:
```
"distributed" = ["dask", "zict"]
```

I'll have a fresh look at `dask` taking into account this change.